packages/vscode/src/extension.ts

```diff
@@ -84,6 +85,24 @@ export async function activate(context: ExtensionContext): Promise<void> {
     },
   }
 
+  // enable fileWatcher to watch .prisma folder inside node_modules
+  const config: WorkspaceConfiguration = workspace.getConfiguration()
+  const value: string = JSON.stringify(config.get('files.watcherExclude', ''))
+  if (value.includes('**/node_modules/*/**')) {
```

Comment:
I guess it would make more sense to not stringify the object and check for the match in the object keys like

```js
if (value['**/node_modules/*/**']) {
// Copy boolean value
value['**/node_modules/{[^.],?[^p],??[^r],???[^i],????[^s],?????[^m]}*'] = value['**/node_modules/*/**']
// Delete original exclude
delete value['**/node_modules/*/**']
}

```

Reply:
The only issue with this, is that currently VSCode does not even look at the boolean values, just checks what keys are there and ignores those glob patterns as already discussed in this Microsoft issue: https://github.com/microsoft/vscode/issues/869#issue-119744338

Reply:
Here it's to keep the same value (enabled / disabled I guess), anyway `delete value['**/node_modules/*/**']` deletes the original exclude so it's ok.

Reply:
Got it, your suggestion works when using `JSON.parse` before this code block and then `JSON.stringify`  on sending the updated value.